notebooks/community/vertex_endpoints/tf_hub_obj_detection/deploy_tfhub_object_detection_on_vertex_endpoints.ipynb

```diff
@@ -0,0 +1,1329 @@
+{
```

Comment:
<p>Apologies for mentioning 2.8 before, as we don't currently support it with a pre-built container. The notebook and cloud training versions should match.</p><p> </p><p>What about updating the tensorflow requirement above to &gt;= 2.7 and using the 2.7 container here?</p><p> </p><p>https://cloud.google.com/vertex-ai/docs/predictions/pre-built-containers</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/319/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='62'/>